src/views/virtualmachines/details/tabs/details/components/grid/rightGrid/VirtualMachineDetailsRightGridLayout.tsx

```diff
@@ -103,10 +128,23 @@ const VirtualMachineDetailsRightGridLayout: React.FC<VirtualMachineDetailsRightG
         )}
         <VirtualMachineDescriptionItem
           descriptionData={
-            getAnnotation(vm?.spec?.template, VM_WORKLOAD_ANNOTATION) || (
+            vmWorkload ? (
```

Comment:
vmWorkload is a boolean? if so consider change the name to hasworkload or something similar

Reply:
No its not a boolean but it can be undefined if the annotation does not exist

Reply:
i think WORKLOADS_LABELS?.[vmWorkload] || vmWorkload ,  can work here , without the tannery, please check